### DISCUSSION TITLE
您好大佬，请问  ”模型计算损失时会忽略标记"##回答:"之前的部分“ 这一块在您哪部分代码实现的？我好像没找到，感谢！！

### DISCUSSION THREAD

**Discussion by jackfsuia** [charent/Phi2-mini-Chinese#10] [Q&A] [answered]
您好大佬，请问  ”模型计算损失时会忽略标记"##回答:"之前的部分“ 这一块在您哪部分代码实现的？我好像没找到，感谢！！

**Comment by charent** [comment, marked as answer]
这个是` DataCollatorForCompletionOnlyLM`实现的（`from trl import DataCollatorForCompletionOnlyLM`），你可以看下源码。直接看`attention_mask`也行。

**Reply by jackfsuia** [reply]
学到了，感谢！
